app/controllers/taxons_controller.rb

```diff
@@ -93,6 +103,10 @@ def destroy_flash_message(response_code)
     end
   end
 
+  def path_prefixes_for_select
+    Theme.taxon_path_prefixes
```

Comment:
It used to be, but `taxonomy_editing_spec.rb` now contains tests that rely on a valid prefix, so changing them may cause confusing failures.